server/src/jobs/auto-discovery.cpp

```diff
@@ -312,8 +312,7 @@ void AutoDiscovery::scan(AutoDiscovery* autoDiscovery, const std::string& ipAddr
                         logInfo("Found asset with protocol/port/credential ({}/{}/{}) for {}", elt.protocol, elt.port, doc, ipAddress);
 
                         auto getAssetStatus = [autoDiscovery]() -> uint {
-                            logTrace("autoDiscovery->isDeviceCentricView()={}", autoDiscovery->isDeviceCentricView());
-                            return autoDiscovery->isDeviceCentricView() ?
+                            return autoDiscovery->m_params.aux.status.value() == "active" ?
```

Comment:
We need to make sure that when activation is failing due to not enough credits, the asset are still inserted in instance with a disable status.